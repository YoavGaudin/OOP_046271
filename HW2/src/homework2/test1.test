# Test empty graph
CreateGraph A
ListNodes A
CreateNode n1 5

# Test no children
AddNode A n1
ListChildren A n1
ListNodes A