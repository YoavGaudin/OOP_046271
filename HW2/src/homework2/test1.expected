# Test empty graph
created graph A
A contains:
created node n1 with cost 5

# Test no children
added node n1 to A
the children of n1 in A are:
A contains: n1
